src/python/grpcio_tests/tests/unit/_auth_context_test.py

```diff
@@ -140,6 +154,57 @@ def testSecureClientCert(self):
         self.assertSequenceEqual([b'*.test.google.com'],
                                  auth_ctx['x509_common_name'])
 
+    def _do_one_shot_client_rpc(self, channel_creds, channel_options, port,
+                                expect_session_reused):
+        channel = grpc.secure_channel(
+            'localhost:{}'.format(port), channel_creds, options=channel_options)
+        response = channel.unary_unary(_UNARY_UNARY)(_REQUEST)
+        auth_data = pickle.loads(response)
+        self.assertEqual(expect_session_reused,
+                         auth_data[_AUTH_CTX]['ssl_session_reused'])
+
+    def testSSLSessionCacheLRU(self):
```

Comment:
Should this feature be tested in a different test class or different test module? It looks like it shares some stuff with auth context, but does it share enough?

Reply:
I've pared the test in`_auth_context_test.py` down to the parts relevant to this file, and moved the others to a separate `_session_cache_test.py`.